src/cdk/table/table-module.ts

```diff
@@ -45,7 +46,7 @@ const EXPORTED_DECLARATIONS = [
 
 @NgModule({
   exports: EXPORTED_DECLARATIONS,
-  declarations: EXPORTED_DECLARATIONS
-
+  declarations: EXPORTED_DECLARATIONS,
+  imports: [ScrollingModule]
```

Comment:
I'm worried that adding this may increase payload size of the table by bringing in incidental stuff. Can we confirm that this doesn't happen?

@devversion can you point to an example of a test here to verify this?

Reply:
Adding this module has no significant impact. This is because the scrolling module, and its transitive imports do not specify explicit non treeshakeable providers or a bootstrap component.

Consumers of the `CdkTableModule` will have to pay the costs of the `ScrollingModule` class with module and injector definitions. Also transitive modules such as `CdkScrollableModule`, `PlatformModule` and `BidiModule` are brought in.

Neither of these transitive modules causes actual components/services/directives to be brought in (unless explicitly used), so the impact is rather little, except for the overhead of these four classes being retained in the JS bundle. e.g.

```js
let ScrollingModule = /*@__PURE__*/ (() => {
    class ScrollingModule {
    }
    ScrollingModule.ɵmod = /*@__PURE__*/ ɵɵdefineNgModule({ type: ScrollingModule });
    ScrollingModule.ɵinj = /*@__PURE__*/ ɵɵdefineInjector({ factory: function ScrollingModule_Factory(t) { return new (t || ScrollingModule)(); }, imports: [[
                BidiModule,
                PlatformModule,
                CdkScrollableModule
            ], BidiModule, CdkScrollableModule] });
    return ScrollingModule;
})();
```

In a basic app (minified) using `CdkTable`, this resulted in an increase of 458 bytes (i.e. 0.36% increase).

Here is an example of a size test for as baisc app w/ `CdkTable`, and the noticeable change when we add `ScrollingModule` to the CDK table module. See latest two commits in: https://github.com/devversion/material2/commits/test/size-test-table.

We should be able to add this test to this PR, if we going to proceed with using `ScrollingModule` in the CDK table module.

Reply:
Ack, thanks for checking that!